lib/rouge/lexers/kotlin.rb

```diff
@@ -56,10 +58,7 @@ class Kotlin < RegexLexer
           groups Keyword::Declaration, Text
           push :property
         end
-        rule %r'(fun)(\s+)' do
-          groups Keyword, Text
-          push :function
-        end
+        rule %r/fun/, Keyword
```

Comment:
Looks good! But I would add a `\b` at the end to make sure it doesn't match `funny` for example.

Reply:
Oh! I've misunderstood it because there are too many /keyword/ there.

Fixed in f798dd4